examples/app.py

```diff
@@ -25,36 +25,70 @@
 
 """Minimal Flask application example for development.
 
+For this example the access control is disabled.
+
 Run example development server:
 
 .. code-block:: console
 
-   $ cd examples
-   $ flask -a app.py db init
-   $ flask -a app.py db create
-   $ flask -a app.py load_fixture
-   $ flask -a app.py --debug run
-"""
+    $ cd examples
+    $ flask -a app.py db init
+    $ flask -a app.py db create
+    $ flask -a app.py fixtures records
+    $ flask -a app.py --debug run
+
+Try to get some records
+
+.. code-block:: console
+
+    $ curl -XGET http://localhost:5000/records/1
+    $ curl -XGET http://localhost:5000/records/2
+    $ curl -XGET http://localhost:5000/records/3
+    $ curl -XGET http://localhost:5000/records/4
+    $ curl -XGET http://localhost:5000/records/5
+    $ curl -XGET http://localhost:5000/records/6
+    $ curl -XGET http://localhost:5000/records/7
+
+Then search for existing records
+
+    $ curl -v -XGET 'http://localhost:5000/records/?size=3'
+    $ curl -v -XGET 'http://localhost:5000/records/?size=2&page=3'
+    $ curl -v -XGET 'http://localhost:5000/records/?q=awesome'
+"""  # noqa
```

Comment:
Sorry I forgot to remove that. At some point there where lines in the documentation which had more 80 characters. Rather than making the doc less readable I had added a `#noqa`.

This will be fixed in the access control pull request as it comes from there. This PR contains only the two last commits.